### PR TITLE
Add token_address

### PR DESCRIPTION
**Pull request by hagaetc**
Hope I got the formatting right :)

I've checked that:

* [ ] the query produces the intended results
* [ ] the folder name matches the schema name
* [ ] the schema name exists in Dune
* [ ] views are prefixed with `view_`, functions with `fn_`.
* [ ] the filename matches the defined view, table or function and ends with .sql
* [ ] each file has only one view, table or function defined  
* [ ] column names are `lowercase_snake_cased`

### CODE REVIEW

File: schema/makermcd/collateral_addresses.sql [hunk 1]
```diff
@@ -2,28 +2,29 @@
...
+\\x4454aF7C8bb9463203b66C816220D41ED7837f44	MakerDAO	\x0000000000085d4780B73119b644AE5ecd22b376  TUSD
+\\xc7e8Cd72BDEe38865b4F5615956eF47ce1a7e5D0	MakerDAO	\xe41d2489571d322189246dafa5ebde1f4699f498  ZRX
+\\x475F1a89C1ED844A08E8f6C50A00228b5E59E4A9	MakerDAO	\xdd974d5c2e2928dea5f71b9825b8b646686bd200  KNC
+\\xA6EA3b9C04b8a38Ff5e224E7c3D6937ca44C0ef9	MakerDAO	\x0f5d2fb29fb7d3cfee444a200298f468908cc942  Mana
+\\x0Ac6A1D74E84C2dF9063bDDc31699FF2a2BB22A2	MakerDAO	\xdac17f958d2ee523a2206206994597c13d831ec7  USDT
+\\x7e62B7E279DFC78DEB656E34D6a435cC08a44666	MakerDAO	\x8e870d67f660d95d5be530380d0ec0bd388289e1  PAXUSD
+\\xBEa7cDfB4b49EC154Ae1c0D731E4DC773A3265aA	MakerDAO	\xc00e94cb662c3520282e6f5717214004a7f26888  COMP
+\\x6C186404A7A238D3d6027C0299D1822c1cf5d8f1	MakerDAO	\xbbbbca6a901c926f240b89eacb641d8aec7aeafd  LRC
+\\xdFccAf8fDbD2F4805C174f856a317765B49E4a50	MakerDAO	\x514910771af9ca656af840dff83e8264ecf986ca  LINK
+\\xBF72Da2Bd84c5170618Fbe5914B0ECA9638d5eb5	MakerDAO	\x2260fac5e5542a773aa44fbcfedf7c193bc2c599  WBTC
```
Comment (mewwts) [inline schema/makermcd/collateral_addresses.sql:27]: This new column should be `\\x`-prefixed like the first column

File: schema/makermcd/collateral_addresses.sql [hunk 2]
```diff
@@ -2,28 +2,29 @@
...
-\\x0Ac6A1D74E84C2dF9063bDDc31699FF2a2BB22A2	MakerDAO	USDT
-\\x7e62B7E279DFC78DEB656E34D6a435cC08a44666	MakerDAO	PAXUSD
-\\xBEa7cDfB4b49EC154Ae1c0D731E4DC773A3265aA	MakerDAO	COMP
-\\x6C186404A7A238D3d6027C0299D1822c1cf5d8f1	MakerDAO	LRC
-\\xdFccAf8fDbD2F4805C174f856a317765B49E4a50	MakerDAO	LINK
-\\xBF72Da2Bd84c5170618Fbe5914B0ECA9638d5eb5	MakerDAO	WBTC
+COPY makermcd.collateral_addresses(address, project, token_address, details) FROM stdin;
+\\x2f0b23f53734252bda2277357e97e1517d6b042a	MakerDAO    \xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2	WETH A
+\\x08638eF1A205bE6762A8b935F5da9b700Cf7322c	MakerDAO	\xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2  WETH B
+\\x3d0b1912b66114d4096f48a8cee3a56c231772ca	MakerDAO	\0x0d8775f648430679a709e98d2b0cb6250d2887ef BAT
```
Comment (mewwts) [inline schema/makermcd/collateral_addresses.sql:15]: `token_address` here starts with `\0x`, but the correct is `\\x`.

File: schema/makermcd/collateral_addresses.sql (hunk 1 above)
Comment (mewwts) [reply schema/makermcd/collateral_addresses.sql:27]: Something is also off with the indentation of the lines here.

**Review by mewwts — CHANGES_REQUESTED** [review]

File: schema/makermcd/collateral_addresses.sql (hunk 1 above)
Comment (hagaetc) [reply schema/makermcd/collateral_addresses.sql]: Seems like the indentation almost always get's messed up for me... Might need to start editing outside Github UI IDK

Comment (mewwts) [reply schema/makermcd/collateral_addresses.sql]: Please do, because this generates a lot of extra work ;p

**Comment by mewwts** [comment]
Merged here https://github.com/duneanalytics/abstractions/commit/5f75f4673702eeeb3ea9a4a8805a92dd1017a41f

**Pull request closed without merging**